### DISCUSSION TITLE
Can't see mp4 files

### DISCUSSION THREAD

**Discussion by nivlem1o1** [QL-Win/QuickLook#1416] [Q&A]
It doesn't support mp4 files? when i was trying to open up an mp4 file, i can only hear the audio; otherwise, its just showed a blank screen

https://github.com/QL-Win/QuickLook/assets/166381396/28b19bb5-ad30-4a3b-ad3d-1a94f04b61b2

**Comment by travibe** [comment]
most likely video codec not compatible with quicklook. I have this problem with MP4 files encoded in AV1 video codec. No idea if dev plans on including this in future versions

**Reply by emako** [reply]
Latest Nightly Build might solve your issues? https://github.com/QL-Win/QuickLook/releases
